pandas/io/common.py

```diff
@@ -169,7 +169,7 @@ def _stringify_path(filepath_or_buffer):
 
 
 def get_filepath_or_buffer(filepath_or_buffer, encoding=None,
-                           compression=None):
+                           compression=None, mode='rb'):
```

Comment:
Add this to the Parameters section of the docstring, and note that it's only really used for S3 files.